packages/forms/signals/src/api/field_directive.ts

```diff
@@ -61,6 +62,10 @@ export const FIELD = new InjectionToken<Field<unknown>>(
 })
 export class Field<T> implements ɵControl<T> {
   private readonly injector = inject(Injector);
+  private options = inject(SIGNAL_FORMS_CONFIG, {optional: true});
+  readonly classes = Object.entries(this.options?.classes ?? {}).map(
```

Comment:
nit: config/options?

```suggestion
  private config = inject(SIGNAL_FORMS_CONFIG, {optional: true});
  readonly classes = Object.entries(this.config?.classes ?? {}).map(
```